### PR TITLE
fixed cstyle issues

### PR DESCRIPTION
**Pull request by kn-neelalohitha**
signed off by kn-neelalohitha kn.neelalohitha@gmail.com

### CODE REVIEW

File: src/istgt_iscsi_param.c [hunk 1]
```diff
@@ -64,14 +64,14 @@ istgt_iscsi_param_find(ISCSI_PARAM *params, const char *key)
...
 	if (params == NULL || key == NULL)
-		return NULL;
+		return (NULL);
 	for (param = params; param != NULL; param = param->next) {
-		if (param->key != NULL && param->key[0] == key[0]
-			&& strcasecmp(param->key, key) == 0) {
-			return param;
+		if (param->key != NULL && param->key[0] == key[0] &&
+		    strcasecmp(param->key, key) == 0) {
+			return (param);
```
Comment (utkarshmani1997) [inline src/istgt_iscsi_param.c:71]: fix indentation

**Review by utkarshmani1997 — COMMENTED** [review]
changes are good. Have provided the comment to fix indentation

File: src/istgt_iscsi_param.c [hunk 2]
```diff
@@ -81,39 +81,40 @@ istgt_iscsi_param_del(ISCSI_PARAM **params, const char *key)
...
 int
-istgt_iscsi_param_add(ISCSI_PARAM **params, const char *key, const char *val, const char *list, int type)
+istgt_iscsi_param_add(ISCSI_PARAM **params, const char *key, const char *val,
+    const char *list, int type)
 {
 	ISCSI_PARAM *param, *last_param;
 
 	ISTGT_TRACELOG(ISTGT_TRACE_DEBUG, "add %s=%s, list=[%s], type=%d\n",
-				   key, val, list, type);
+			key, val, list, type);
```
Comment (vishnuitta) [inline src/istgt_iscsi_param.c]: it should be 4 spaces under ISTGT_TRACELOG

**Comment by utkarshmani1997** [comment]
Hi @kn-neelalohitha , rebase your branch with the latest changes. It is conflicting with the existing ones.

**Comment by vishnuitta** [comment]
Sorry to say that this issue has been fixed with changes from PR#100. Hence, closing this PR.

**Pull request closed without merging**